site/static/js/carousel-video-mutation-observer.js

```diff
@@ -0,0 +1,42 @@
+export default function carouselVideoPauseOnChange() {
+  // load the first video
+  let s = document.querySelector('.s72-carousel-item.current video source');
```

Comment:
probably should rename the function to something more meanful, and potentially pull out this play first video to a different file perhaps